backend/src/main/java/com/bakdata/conquery/models/config/ArrowResultProvider.java

```diff
@@ -18,13 +20,17 @@
 import io.dropwizard.jersey.DropwizardResourceConfig;
 import lombok.Data;
 import lombok.SneakyThrows;
-import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.glassfish.jersey.internal.inject.AbstractBinder;
 
 @Data
 @CPSType(base = ResultRendererProvider.class, id = "ARROW")
 public class ArrowResultProvider implements ResultRendererProvider {
 	private boolean hidden = true;
 
+	@Valid
+	@NotNull
+	private ArrowConfig config = new ArrowConfig();
```

Comment:
cool, macht Sinn!

Reply:
Du könntest hier auch mit dem Gedanken spielen es mit JsonUnwrapped zu lesen, dass du das ganze flattest?

Reply:
JsonUnwrapped tickelt dich immer mal wieder :D soweit ich weiß geht unwrap aber nur in die serialisierungs Richtung. Bei Objekten in der Config brauchen wir aber beiderichtungen, weil wir die Config in den Test auch kopieren.